ui/widgets/nuxeo-selectivity.js

```diff
@@ -6784,6 +6784,7 @@ typedArrayTags[weakMapTag] = false;
 
           :host {
             overflow: auto;
+            overflow-x: hidden;
```

Comment:
Before we changed overflow to auto, what was the value?

Reply:
We didn't have anything. The `overflow: auto` was introduced by me in the context of [this ticket](https://jira.nuxeo.com/browse/ELEMENTS-1230). In this situation, we can go with the current suggestion or: 
```suggestion
            overflow-y: auto;
            overflow-x: hidden;
```
Because of this:
> Computed value:  as specified, except with visible/clip computing to auto/hidden respectively if one of overflow-x or overflow-y is neither visible nor clip
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-x